apps/opencs/view/render/terraintexturemode.hpp

```diff
@@ -18,22 +19,34 @@
 #include "../../model/world/landtexture.hpp"
 #endif
 
+#include "terrainselection.hpp"
```

Comment:
I think it should be possible to forward-declare TerrainSelection instead of including this header.

Reply:
I tried this, then added missing include osg/Group, but this still results in some hefty compile errors. "/usr/include/c++/8.2.1/bits/unique_ptr.h:79:16: error: invalid application of ‘sizeof’ to incomplete type ‘CSVRender::TerrainSelection’" Doesn't ring any obvious bells how to fix this.

Reply:
OK, never mind, it can't be done cleanly.